### ISSUE TITLE
Dragging doesn't work on mac+chrome

### ISSUE THREAD

**Issue by blackhole12** [blackhole12/PonyClicker#80]
Dragging doesn't work on Mac OSX or iOS chrome browsers for some reason. I have not found a system to try and repro this on yet.

**Comment by ngyikp** [comment]
You mean dragging and spinning the big pony wheel around? Works fine on my Chrome on Mac OS.

**Comment by blackhole12** [comment]
Yes, several people have reported this issue, and all of them are on chrome on a mac or iphone.

**Comment by blackhole12** [comment]
I have confirmed this bug on iOS with chrome but have no way of debugging it because I don't have an iPhone. If another developer does, maybe they can investigate this issue?
